spec/features/5_photos_spec.rb

```diff
@@ -1,120 +1,126 @@
+      end
 
-    visit "/photos"
+      @user_1.photos.each do |photo|
+        expect(page).not_to have_link(nil, href: "/delete_photo/#{photo.id}")
+      end
+    end
 
-    expect(page).to have_css(".panel")
-  end
+    scenario "in /photos, Bootstrap panels used", points: 1 do
+      user = FactoryGirl.create(:user)
+      photo = FactoryGirl.create(:photo, :user_id => user.id)
+      login_as(user, :scope => :user)
 
-  scenario "in /photos, Font Awesome fa-plus icon used (and 'Photos' h1 tag isn't)", points: 1 do
-    user = FactoryGirl.create(:user)
-    photo = FactoryGirl.create(:photo, :user_id => user.id)
-    login_as(user, :scope => :user)
+      visit "/photos"
 
-    visit "/photos"
+      expect(page).to have_css(".panel")
+    end
 
-    expect(page).to have_css(".fa-plus")
-    expect(page).not_to have_selector("h1", text: "Photos")
-  end
+    scenario "in /photos, Font Awesome fa-plus icon used (and 'Photos' h1 tag isn't)", points: 1 do
+      user = FactoryGirl.create(:user)
+      photo = FactoryGirl.create(:photo, :user_id => user.id)
+      login_as(user, :scope => :user)
 
-  scenario "/photos displays per-photo username, photo, and time elapsed", points: 1, hint: h("time_in_words") do
-    user_1 = FactoryGirl.create(:user, :username => "user_1", :email => "1@m.com")
-    user_2 = FactoryGirl.create(:user, :username => "user_2", :email => "2@m.com")
-    user_3 = FactoryGirl.create(:user, :username => "Three", :email => "three@m.com")
-    photo_1 = FactoryGirl.create(:photo, :user_id => user_1.id)
-    photo_2 = FactoryGirl.create(:photo, :user_id => user_2.id)
-    login_as(user_3, :scope => :user)
+      visit "/photos"
 
-    visit "/photos"
+      expect(page).to have_css(".fa-plus")
+      expect(page).not_to have_selector("h1", text: "Photos")
+    end
 
-    expect(page).to have_content("#{photo_1.user.username}")
-    expect(page).to have_css("img[src*='#{photo_1.image}']")
-    expect(page).to have_content("#{photo_2.user.username}")
-    expect(page).to have_css("img[src*='#{photo_2.image}']")
-    expect(page).to have_content("ago")
-  end
+    scenario "/photos displays per-photo username, photo, and time elapsed", points: 1, hint: h("time_in_words") do
+      login_as(@user_3, :scope => :user)
 
-  scenario "header uses Font Awesome fa-sign-out icon", points: 1 do
-    user = FactoryGirl.create(:user)
-    login_as(user, :scope => :user)
+      visit "/photos"
+      photos = Photo.all
 
-    visit "/photos"
+      photos.each do |photo|
+        expect(page).to have_content("#{photo.user.username}")
+        expect(page).to have_css("img[src*='#{photo.image}']")
+        expect(page).to have_content(ApplicationController.helpers.time_ago_in_words(photo.created_at))
+      end
+    end
 
-    within("nav") {
-      expect(page).to have_css(".fa-sign-out")
-    }
-  end
+    scenario "header uses Font Awesome fa-sign-out icon", points: 1 do
```

Comment:
Let's drop the dependency on and test for Font Awesome, and instead just look for copy: "Sign out"

(The target needs to be modified too.)